workflow-templates/deploy-app-charts.yml

```
@@ -134,14 +130,12 @@ jobs:
 
   feature-test-in-stg:
     needs: deploy-in-stg
-    uses: eeveebank/github-actions-backend/.github/workflows/run-test-suite.yaml@master
+    uses: eeveebank/feature-tests/.github/workflows/run-feature-tests.yaml@master #TODO Use your test repositories workflow file here
     secrets: inherit
     with:
-      repository: eeveebank/feature-tests #TODO Add your application feature tests repository name here
-      ref: master
       tags: "@ExampleTag" #TODO Replace with your test tags
-      accountProvider: pps
-      environment: dev
+      accountProvider: genie #TODO Configure inputs for your tests i.e. ssv does not need accountProvider
+      environment: staging
```

isn't this meant to be 
```suggestion
      environment:`stg`
```

---

think it should be staging for these tests
https://github.com/eeveebank/feature-tests/blob/93b9e271eb3de452283523af6e4f26383279c79a/build.gradle#L96